src/coffee/client.coffee

```diff
@@ -220,9 +226,11 @@ class SphereClient
   #
   # options - An {Object} to configure the client
   constructor: (options = {}) ->
-
     # Private: instance of a {TaskQueue}
-    @_task = options.task or new TaskQueue
+    if options.enableRepeater? and !options.enableRepeater
```

Comment:
Hm not sure if this was intended, but this condition requires the `enableRepeater` option to exist and to be falsy. This means that if the `enableRepeater` option is not provided it would go into the else block. This looks like an unwanted side-effect to me.